code/ATMOSPHERICS/datum_pipeline.dm

```diff
@@ -121,11 +121,7 @@ var/pipenetwarnings = 10
 
 /obj/machinery/atmospherics/components/addMember(obj/machinery/atmospherics/A)
 	var/datum/pipeline/P = returnPipenet(A)
-	for(var/I = 1; I <= parents.len; I++)
-		if(parents["p[I]"] == P)
-			P.addMember(A, src)
-			break
-
+	P.addMember(A, src)

```

Comment:
I don't actually know why I changed that in the first place. This is just restoring it to how it was before my refactor. It works fine.
https://github.com/tgstation/-tg-station/pull/10560/files#diff-5ab0e677d41168f5f7061e5ec68ab6f9L124